HHAuto.user.js

```diff
@@ -581,7 +587,24 @@ function setLastPageCalled(inPage)
 var proceedQuest = function () {
     //logHHAuto("Starting auto quest.");
     // Check if at correct page.
-    if (getPage() !== getHHScriptVars("pagesIDQuest")) {
+    let page = getPage();
+    let mainQuestUrl = getHHVars('Hero.infos.questing.current_url');
+    let doMainQuest = getStoredValue("HHAuto_Setting_autoQuest") === "true" && !mainQuestUrl.includes("world");
+    if (!doMainQuest && page === 'side-quests' && getHHScriptVars("isEnabledSideQuest",false) && getStoredValue("HHAuto_Setting_autoSideQuest") === "true") {
+        var quests = $('.side-quest:has(.slot) .side-quest-button');
+        if (quests.length > 0) {
+            logHHAuto("Navigating to side quest.");
+            gotoPage(quests.attr('href'));
+        }
+        else {
+            logHHAuto("All quests finished, turning off AutoQuest!");
+            setStoredValue("HHAuto_Setting_autoQuest", false);
```

Comment:
we should only disable sidequest in this case right ?

Reply:
no if (!mainQuestUrl.includes("world")) == false then !doMainQuest = true

Reply:
@react31 you're right, re fixing it now
thanks for the help